eng/SharedFramework.External.props

```diff
@@ -56,9 +56,6 @@
     <ExternalAspNetCoreAppReference Include="System.Security.Cryptography.Xml"                             Version="$(SystemSecurityCryptographyXmlPackageVersion)" />
     <ExternalAspNetCoreAppReference Include="System.Threading.Channels"                                    Version="$(SystemThreadingChannelsPackageVersion)" />
 
-    <!-- Dependencies from dotnet/core-setup -->
-    <ExternalAspNetCoreAppReference Include="Microsoft.Extensions.DependencyModel"                         Version="$(MicrosoftExtensionsDependencyModelPackageVersion)" />
```

Comment:
cc @natemcmaster

Reply:
Can you update this list? https://github.com/aspnet/AspNetCore/issues/3755